doc/content/verification_and_validation/val-2f.md

```diff
@@ -0,0 +1,100 @@
+
+The TDS process is simulated using TMAP8 in a 1D tungsten sample with a thickness of 0.8 mm. The TDS simulation consisted of three phases: implantation, resting, and desorption.
+
+During the charging phase, deuterium is continuously implanted into the tungsten sample over a period of 72 hours. The temperature is maintained at 370 K throughout this phase. The surface is exposed to a constant flux of $\phi=5.79\times 10^{19}$ atoms/m$^2$/s, corresponding to a total fluence of $1.5\times 10^{25}$ atoms/m$^2$. The implantation profile follows a Gaussian distribution centered at the mean implantation depth of $R_p=0.7$ nm, with a standard deviation of $\sigma = 0.5$ nm :
+
+\begin{equation}
+    S(x) = \frac{1}{\sigma \sqrt{2\pi}} \exp\left( -\frac{(x - R_p)^2}{2\sigma^2} \right) \cdot \phi_{\text{surface}}(t)
+\end{equation}
+
+where the surface flux function is given by:
+
+\begin{equation}
+    \phi_{\text{surface}}(t) =
+    \begin{cases}
+        \phi, & t < 72 h \\
+        0, & \text{otherwise}
+    \end{cases}
+\end{equation}
+
+The implantation distribution is illustrated in [val-2f_implantation_distribution]. It highlights the need for mesh refinement on the order of $\sigma$ in the implantation region to accurately resolve the profile and capture the sharp gradient of the source term.
+
+!media comparison_val-2f.py
+    image_name=val-2f_implantation_distribution.png
+    style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+    id=val-2f_implantation_distribution
+    caption=Deuterium implantation.
+
+After the implantation phase, the system enters the cooldown phase, lasting 12 hours. During this period, the sample temperature is rapidly reduced from 370 K to 295 K. No additional deuterium is introduced during this phase, meaning the source term is set to zero.
+
+The final stage of the simulation is the desorption phase, during which the sample is gradually heated from 300 K to 1000 K at a constant rate of $\beta = 0.05$ K/s.
+
+The general form of the governing diffusion equation for the deuterium concentration $C(x,t)$ in tungsten is given by:
+
+\begin{equation}
+    \frac{\partial C}{\partial t} = \nabla \cdot \left( D \nabla C \right) + S(x, t)
+\end{equation}
+
+The diffusion coefficient $D(T)$ follows an Arrhenius-type dependency on temperature:
+
+\begin{equation}
+    D(T) = D_0 \exp\left(-\frac{E_D}{k_B T}\right)
+\end{equation}
+
+On the left boundary, deuterium recombines into gas. Assuming the recombination process is at steady state, it can be described by the following surface flux:
+
+\begin{equation}
+    J = 2 A K_r C^2
+\end{equation}
+
+where $J$ represents the recombination flux out of the left sample side, $A$ is the area that side, and $K_r$ is the deuterium recombination coefficient. The coefficient of 2 accounts for the fact that 2 deuterium atoms combine to form one D$_2$ molecule. The flux out of the right sample side is assumed to be zero.
```

Comment:
```suggestion
where $J$ represents the recombination flux out of the left sample side, $A$ is the area that side, and $K_r$ is the deuterium recombination coefficient. The coefficient of 2 accounts for the fact that 2 deuterium atoms combine to form one D$_2$ molecule.
```
Both surfaces should be treated the same here. Moreover, the concentration was fixed to 0, not the flux.